system/GD32F30x_firmware/GD32F30x_standard_peripheral/Include/gd32f30x_ctc.h

```diff
@@ -97,7 +97,6 @@ OF SUCH DAMAGE.
 #define CTL1_REFSEL(regval)                              (BITS(28,29) & ((uint32_t)(regval) << 28))
 #define CTC_REFSOURCE_GPIO                               CTL1_REFSEL(0)               /*!< GPIO is selected */
 #define CTC_REFSOURCE_LXTAL                              CTL1_REFSEL(1)               /*!< LXTAL is selected */
-#define CTC_REFSOURCE_USB_SOF                            CTL1_REFSEL(2)               /*!< USBD_SOF or USBFS_SOF is selected */

```

Comment:
Would it be possible for us to locally revert the change that breaks the SOF CTC source?

Reply:
Yeah, we could do that. Do you want that added to this PR or a separate one?

I know it appears to work, at least on the one piece of hardware that I have access to. Unfortunately, we still don't know why it was silently removed. We might not want to use it if it causes some non-obvious catastrophic behavior some of the time.